internal/deepdoc/native/find_contours_proto.py

```
@@ -0,0 +1,113 @@
+                    break
+            if len(contour) >= 3:
+                contours.append(np.array(contour, np.int32).reshape(-1, 1, 2))
+                nbd += 1
+            # mark start visited regardless
+            visited[r, c] = nbd
+    return contours
+
+
+if __name__ == "__main__":
+    import json
+    py = json.load(open("/tmp/py_seg.json"))
```

Ruff (0.16.2)

[error] Probable insecure usage of temporary file or directory: "/tmp/py_seg.json"

(S108)